rocAL/rocAL_pybind/example/tf_mnistTrainingExample/README.md

```diff
@@ -1,35 +0,0 @@
-## Tensorflow MNIST digit classification example with rocAL pipeline
```

Comment:
please add the readme back

Reply:
The mnist example will be added as a part of mini trainings PR.